ivy_tests/test_ivy/test_frontends/test_jax/test_random.py

```diff
@@ -1518,3 +1518,58 @@ def call():
     for u, v in zip(ret_np, ret_from_np):
         assert u.dtype == v.dtype
         assert u.shape == v.shape
+
+
+# @pytest.mark.xfail
+@handle_frontend_test(
+    fn_tree="jax.random.logistic",
+    dtype_key=helpers.dtype_and_values(
+        available_dtypes=["uint32"],
+        min_value=0,
+        max_value=2000,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+        max_dim_size=2,
+    ),
+    shape=helpers.get_shape(allow_none=False, min_num_dims=1, min_dim_size=1),
+    dtype=helpers.get_dtypes("float", full=False),
+    test_with_out=st.just(False),
+)
+def test_jax_logistic(
+    *,
+    dtype_key,
+    shape,
+    dtype,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+):
+    input_dtype, key = dtype_key
+
+    def call():
+        helpers.test_frontend_function(
+            input_dtypes=input_dtype,
+            frontend=frontend,
+            test_flags=test_flags,
+            backend_to_test=backend_fw,
+            fn_tree=fn_tree,
+            test_values=False,
+            key=key[0],
+            shape=shape,
+            dtype=dtype[0],
+        
+        )
+
+    ret = call()
+
+    if not ivy.exists(ret):
+        return
+
+    ret_np, ret_from_np = ret
```

Comment:
Why have you added it here? The value comparison of the groundtruth backend and frontend framework result is being carried out in the test_frontend_function. I don't think you need to implement it over here